rlberry/utils/tests/test_sb_agent.py

```diff
@@ -0,0 +1,37 @@
+from stable_baselines3 import A2C
+
+from rlberry.envs import gym_make
+from rlberry.utils.sb_agent import StableBaselinesAgent
+from rlberry.utils.check_agent import (
+    check_agent_manager,
+    check_seeding_agent,
+    check_save_load,
+)
+
+
+def _sb3_check_rl_agent(agent_cls, env, init_kwargs=None):
+    """
+    Check that an RL agent can be instantiated and fit.
+
+    Note
+    ----
+    Does not check that the fit is additive, because that doesn't seem to be
+    always the case with StableBaselines3.
+    """
+    check_agent_manager(agent_cls, env, init_kwargs=init_kwargs)
```

Comment:
Maybe we could increase the `verbose` in the `init_kwargs` here to make sure that the writer wrapper is tested?

Reply:
I'm taking a look into this. There's something wrong with the loading when I set `verbose=1`.